modules/core/shared/src/main/scala/gem/enum/MagnitudeBand.scala

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+package enum
+
+import cats.syntax.eq._
+import cats.instances.string._
+import gem.util.Enumerated
+
+/**
+ * Enumerated type for magnitude band.
+ * @group Enumerations (Generated)
+ */
+sealed abstract class MagnitudeBand(
+  val tag: String,
+  val shortName: String,
+  val longName: String,
+  val center: gem.math.Wavelength,
+  val width: gem.math.Wavelength,
```

Comment:
I wonder if it makes sense to say that the width is a wavelength.

Reply:
You are right, it looks like wavelength but it isn't.